cmake/AdaptSimSYCL.cmake

```diff
@@ -0,0 +1,45 @@
+add_library(SYCL::SYCL INTERFACE IMPORTED GLOBAL)
```

Comment:
Why this file is named `AdaptSimSYCL` and not `SimSYCL`?

Reply:
We've had `Find*` and `Adapt*` files for a while now, it has nothing to do with AdaptiveCpp - the corresponding file for AdaptiveCpp in #874 is called `AdaptAdaptiveCpp` ;-).